dev/benchmarks/macrobenchmarks/lib/src/cull_opacity.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+class CullOpacityPage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _CullOpacityPageState();
+}
+
+class _CullOpacityPageState extends State<CullOpacityPage> with SingleTickerProviderStateMixin {
+  Animation<double> _offsetY;
+  AnimationController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _controller = AnimationController(vsync: this, duration: Duration(seconds: 2));
+    _offsetY = Tween<double>(begin: 0, end: -1000.0).animate(_controller)..addListener((){
+      setState((){});
```

Comment:
Nit: `() {}` (add a space between the two pairs of brackets)

Reply:
Done.